source/openapi-admin-v3.yaml

```diff
@@ -2646,16 +2654,21 @@ paths:
         content:
           application/json:
             schema:
-              $ref: "#/components/schemas/PrivateEndpoint"
+              $ref: "#/components/schemas/PrivateEndpointCreate"
```

Comment:
`_id` is actually not a required field because it's encoded in the path param: `api/admin/v3.0/groups/:groupId/apps/:appId/security/private_endpoints/:endpointId`

So any `_id` passed into the body gets ignored.